app/src/standard/java/mihon/core/firebase/Firebase.kt

```diff
@@ -14,7 +14,7 @@ object Firebase {
             FirebaseAnalytics.getInstance(context).setAnalyticsCollectionEnabled(enabled)
         }.launchIn(scope)
         preference.crashlytics().changes().onEach { enabled ->
-            FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(enabled)
+            FirebaseCrashlytics.getInstance().isCrashlyticsCollectionEnabled = enabled
```

Comment:
It's the same thing

Reply:
yes, same, just that ktlint or Android Studio suggests using that form. Should I revert it?

Reply:
Yes please